docs/data-sources/mongodb_instance.md

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "MongoDB®"
+page_title: "Scaleway: scaleway_mongodb_instance"
+---
+
+# scaleway_mongodb_instance
+
+Gets information about a MongoDB® Instance.
+
+For further information refer to the Managed Databases for MongoDB® [API documentation](https://developers.scaleway.com/en/products/mongodb/api/)
+
+## Example Usage
+
+```hcl
+# Get info by name
+data "scaleway_mongodb_instance" "my_instance" {
+  name = "foobar"
+}
+
+# Get info by instance ID
+data "scaleway_mongodb_instance" "my_instance" {
+  instance_id = "11111111-1111-1111-1111-111111111111"
+}
+
+# Get other attributes
+output "mongodb_version" {
+  description = "Version of the MongoDB instance"
+  value = data.scaleway_mongodb_instance.my_instance.version
+}
+```
+
+## Argument Reference
+
+- `name` - (Optional) The name of the MongoDB® instance.
+
+- `instance_id` - (Optional) The MongoDB® instance ID.
+
+  -> **Note** You must specify at least one: `name` or `instance_id`.
+
+- `project_id` - (Optional) The ID of the project the MongoDB® instance is in. Can be used to filter instances when using `name`.
+
+- `region` - (Defaults to [provider](../index.md#region) `region`) The [region](../guides/regions_and_zones.md#zones) in which the MongoDB® Instance exists.
+
+- `organization_id` - (Defaults to [provider](../index.md#organization_id) `organization_id`) The ID of the organization the MongoDB® instance is in.
+
+## Attributes Reference
+
+In addition to all above arguments, the following attributes are exported:
+
+- `id` - The ID of the MongoDB® Instance.
```

Comment:
Will this ID still be present?

Reply:
That's normal, id is always exported, right?